packages/lesswrong/components/ea-forum/voting-portal/VotingPortalHeader.tsx

```diff
@@ -0,0 +1,293 @@
+      color: theme.palette.givingPortal.homepageHeader.main,
+      "&:hover": {
+        backgroundColor: theme.palette.givingPortal.homepageHeader.light1,
+      },
+    },
+    "& .EAButton-greyContained": {
+      backgroundColor: theme.palette.givingPortal.homepageHeader.secondary,
+      color: theme.palette.givingPortal.homepageHeader.light3,
+      "&:hover": {
+        backgroundColor: `${theme.palette.givingPortal.homepageHeader.secondaryDark} !important`,
+      },
+    },
+  },
+  hideLgUp: {
+    [theme.breakpoints.up('lg')]: {
+      display: "none !important"
+    }
+  },
+  hideMdDown: {
+    [theme.breakpoints.down('md')]: {
+      display: "none !important"
+    }
+  },
+  hideSmDown: {
+    [theme.breakpoints.down('sm')]: {
+      display: "none !important",
+    },
+  },
+  hideXsDown: {
+    [theme.breakpoints.down('xs')]: {
+      display: "none !important",
+    },
+  },
+  hideMdUp: {
+    [theme.breakpoints.up('md')]: {
+      display: "none !important",
+    },
+  },
+  toolbarGivingSeason: {
+    zIndex: theme.zIndexes.spotlightItem,
+    justifyContent: "space-between",
+  },
+  siteLogoGivingSeason: {
+    width: 34,
+    [theme.breakpoints.down("sm")]: {
+      width: 30,
+    },
+  },
+  titleLinkGivingSeason: {
+    color: theme.palette.givingPortal.homepageHeader.light4,
+  },
+  navigationSteps: {
+    fontFamily: theme.palette.fonts.sansSerifStack,
+    color: theme.palette.text.alwaysWhite,
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    gap: '24px',
+    fontSize: 16,
+    fontWeight: 500,
+    whiteSpace: 'nowrap',
+    [theme.breakpoints.down('md')]: {
+      gap: '12px',
+    }
+  },
+  activeStepLink: {
+    textUnderlineOffset: '6px',
+    textDecoration: 'underline',
+    '&:hover': {
+      textDecoration: 'underline',
+    }
+  },
+  disabledStepLink: {
+    opacity: 0.7,
+  }
+});
+
+const socialImageProps: CloudinaryPropsType = {
+  dpr: "auto",
+  ar: "16:9",
+  w: "1200",
+  c: "fill",
+  g: "center",
+  q: "auto",
+  f: "auto",
+};
+
+const VOTING_STEPS = [
+  {
+    label: '1. Select candidates',
+    href: '/voting-portal/select-candidates',
+  },
+  {
+    label: '2. Compare',
+    href: '/voting-portal/compare',
+  },
+  {
+    label: '3. Allocate votes',
+    href: '/voting-portal/allocate-votes',
+  },
+  {
+    label: '4. Submit',
+    href: '/voting-portal/submit',
+  },
+] as const
+
+const VotingPortalHeader = ({
+  searchOpen,
+  hasLogo,
+  unFixed,
+  setUnFixed,
+  NavigationMenuButton,
+  RightHeaderItems,
+  HeaderNavigationDrawer,
+  HeaderNotificationsMenu,
+  classes,
+}: {
+  searchOpen: boolean,
+  hasLogo: boolean,
+  unFixed: boolean
+  setUnFixed: (value: boolean) => void,
+  NavigationMenuButton: FC,
+  RightHeaderItems: FC,
+  HeaderNavigationDrawer: FC,
+  HeaderNotificationsMenu: FC,
+  classes: ClassesType,
+}) => {
+  const { Typography, HeadTags } = Components;
+  const { pathname } = useLocation();
+
+  return (
```

Comment:
I'm not sure this is worth making a separate component vs just integrating direcctly into `GivingSeasonHeader` with a `pathname` check (I don't think we should worry about it getting messy for such a short-lived component). My main worry is that there are known bugs with this already which will now need to be fixed in two places (see #8293 for instance which would now need to be copied into this file).